tools/test-backend

```diff
@@ -81,6 +81,7 @@ not_yet_fully_covered = [
     "zerver/worker/base.py",
     "zerver/worker/queue_processors.py",
     "zerver/worker/test.py",
+    "zerver/actions/data_import.py",
```

Comment:
I think it'd be better to use nocoverage comments?

Reply:
I added complete coverage for that file